docs-web/installation/building-ingress-controller-image.md

```diff
@@ -22,17 +22,17 @@ We build the image using the make utility and the provided `Makefile`. Let’s c
 1. Clone the Ingress controller repo:
     ```
     $ git clone https://github.com/nginxinc/kubernetes-ingress/
-    $ git checkout v1.6.1
+    $ cd kubernetes-ingress
+    $ git checkout v1.6.2
     ```
 
 1. Build the image:
     * For NGINX:
       ```
-      $ make clean
       $ make PREFIX=myregistry.example.com/nginx-ingress
       ```
       `myregistry.example.com/nginx-ingress` defines the repo in your private registry where the image will be pushed. Substitute that value with the repo in your private registry.
-      
+
       As the result, the image **myregistry.example.com/nginx-ingress:1.6.0** is built and pushed to the registry. Note that the tag `1.6.0` comes from the `VERSION` variable, defined in the Makefile.
```

Comment:
Do we change 1.6.0 here? Not sure if we did this before 🤔

Reply:
Hmm. We didn't before, but it should be changed.